packages/ui/src/lib/EmptyStatePlaceholder.svelte

```diff
@@ -1,10 +1,27 @@
+<script lang="ts" module>
+	export interface Props {
```

Comment:
Are you using this import somewhere? If not, lets move this into the same `<script lang="ts">` block, i.e. down on line ~15

Reply:
Now it's unfortunately needed for Storybook

Here is an example:
<img width="612" alt="image" src="https://github.com/user-attachments/assets/79ef8b5d-ef27-47e4-ba19-99a0aa00da34">